install/gcp/deployment_manager/istio-cluster.yaml

```diff
@@ -9,12 +9,12 @@ resources:
     gkeClusterName: istio-cluster
     zone: us-central1-a
     initialNodeCount: 4
-    instanceType: n1-standard-1
+    instanceType: n1-standard-2
```

Comment:
why larger instances?

Reply:
the first couple of time i ran istio 1.1, there seemed to be some resource constraints (CPU/memory) errors in GKE.  Tried it again a couple of times, `n1-standard-1` still works. reverted